src/Microsoft.DotNet.Build.Tasks/Targets/partialfacades.targets

```diff
@@ -0,0 +1,141 @@
+         Uses the special partial-facade project.json file to restore the NuGet package containing the partial facade's contract assembly.
+         Filters the list of references returned by the ResolveNuGetPackageAssets task to just the contract assembly and stores it.
+  -->
+  <Target Name="FindPartialFacadeContractReference"
+          Condition="'$(IsPartialFacadeAssembly)' == 'true'"
+          BeforeTargets="FillPartialFacade">
+    
+    <PropertyGroup>
+      <CustomPartialFacadeJsonFile>$(MSBuildProjectDirectory)\partial-facade-packages\project.json</CustomPartialFacadeJsonFile>
+    </PropertyGroup>
+
+    <Error 
+      Condition="!Exists('$(CustomPartialFacadeJsonFile)')"
+      Text="Partial facades require a project.json containing their contract package under src\partial-facade-packages\project.json."
+      />
+
+    <!-- Restore the special partial facade json file, which should contain the contract package to use in GenFacades -->
+    <Exec Command="$(DnuRestoreCommand) &quot;$(CustomPartialFacadeJsonFile)&quot;" StandardOutputImportance="Low" CustomErrorRegularExpression="^Unable to locate .*" />
+
+      <!-- Use new task for project.json -->
+    <ResolveNuGetPackageAssets Condition="'$(ProjectJson)' != ''"
+                               AdditonalProjectJsonFiles="$(CustomPartialFacadeJsonFile)"
+                               Architecture="$(PlatformTarget)"
+                               Configuration="$(Configuration)"
+                               Language="$(Language)"
+                               IngoreLockFile="$(NuGetIngoreLockFile)"
+                               PackageRoot="$(PackagesDir)"
+                               ProjectFile="$(MSBuildProjectFullPath)"
+                               TargetFrameworkMonikers="$(NugetTargetFrameworkMoniker)"
+                               TargetPlatformMonikers="$(TargetPlatformMoniker)"
+                               TransitiveProjectReferences="">
+      <Output TaskParameter="ResolvedReferences" ItemName="PossibleContractRefs" />
+    </ResolveNuGetPackageAssets>
+
+    <ItemGroup>
+      <ResolvedMatchingContract Include="@(PossibleContractRefs)" Condition="'%(FileName)' == '$(AssemblyName)'" />
+    </ItemGroup>
+
+  </Target>
+
+  <!-- FillPartialFacade
+       Inputs:
+         * An "input assembly"
+         * A contract assembly
+         * Seed assemblies
+
+       Fills the "input assembly" by finding types in the contract assembly that are missing from it, and adding type-forwards
+         to those matching types in the seed assemblies. -->
+
+  <!-- Inputs and outputs of FillPartialFacade -->
+  <PropertyGroup Condition="'$(IsPartialFacadeAssembly)' == 'true'">
+    <PartialFacadeAssemblyPath>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</PartialFacadeAssemblyPath>
+    <PartialFacadeSymbols>$(IntermediateOutputPath)$(TargetName).pdb</PartialFacadeSymbols>
+    <GenFacadesInputPath>$(IntermediateOutputPath)PreGenFacades\</GenFacadesInputPath>
```

Comment:
Some recent changes were replacing \ with / to be friendly to unix builds.  Not sure if that applies here.